lib/configproxy.js

```diff
@@ -459,22 +462,47 @@ ConfigurableProxy.prototype.handle_proxy_web = function (req, res) {
     return this.handle_proxy('web', req, res);
 };
 
+function log_response(req, res) {
+    // log function called when any response is finished
+    var code = res.statusCode;
+    var log_f = log.info;
+    if (code >= 400) {
+        log_f = log.warn;
+    } else if (code >= 500) {
+        log_f = log.error;
```

Comment:
@minrk I'm assuming that you wish to log a warning and an error when code is greater than 500.